docs/sources/alerting/meta-monitoring/_index.md

```diff
@@ -0,0 +1,147 @@
+---
+aliases:
+  - meta-monitoring/
+description: Meta monitoring
+keywords:
+  - grafana
+  - alerting
+  - meta-monitoring
+title: Meta monitoring
+weight: 500
+---
+
+# Meta monitoring
+
+Meta monitoring is the process of monitoring your monitoring, and alerting when your monitoring is not working as it should. Whether you use Grafana Managed Alerts or Mimir, meta monitoring is possible both on-premise and in Grafana Cloud.
```

Comment:
Can we help users understand the configuration details to scrape meta-monitoring metrics on their Grafana Cloud instance?
* URL
* authentication parameters
* Anything else needed